modules/reference/pages/properties/broker-properties.adoc

```diff
@@ -6,6 +6,8 @@ Broker configuration properties are applied individually to each broker in a clu
 
 For information on how to edit broker properties, see xref:manage:cluster-maintenance/node-property-configuration.adoc[].
 
+NOTE: Modifying a broker property requires that you restart Redpanda for your changes to take effect. 
```

Comment:
```suggestion
NOTE: All broker property require that you restart Redpanda for any update to take effect.
```

Reply:
@Deflaimun Let's put a similar NOTE on the cluster properties page and the object storage properties page:

- NOTE: Some cluster properties require that you restart Redpanda for any update to be applied. 
- NOTE: Some object storage properties require that you restart Redpanda for any update to be applied. 

And is it correct that no topic properties require restart?